packages/socket-mode/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack/socket-mode",
-  "version": "2.0.3-rc.1",
```

Comment:
I messed [this merge](https://github.com/slackapi/node-slack-sdk/pull/2099/files#diff-6ce92bbc9c794026b6793cbac24c6b968f1e8e11add1bd848675f3eaad0e3702L3) up in #2099!

The current release is `2.0.2` so I think removing the "release candidate" part is alright to bump this to `2.0.3` 🏷️

Reply:
ahhh ok gotcha!! will update 😄